### PR TITLE
Add configuration where necessary to follow FriendsOfCake standards

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

**Comment by josegonzalez** [comment]
Some of these changes are maybe suspect, but I figure they haven't hurt on other projects I've worked on, so I don't think anyone will mind.

We really do need to cleanup the travis:3.x branch and start pushing it's usage out everywhere...

File: .travis.yml [hunk 1]
```diff
@@ -6,34 +6,55 @@ php:
...
     - DEFAULT=1
 
 matrix:
+  allow_failures:
+    - php: hhvm
+
   fast_finish: true
 
   include:
-  - php: 7.0
```
Comment (lorenzo) [inline .travis.yml]: No php 7?

Comment (lorenzo) [inline .travis.yml]: Better said... why use a slower version of php for checking coding standards?

Comment (josegonzalez) [inline .travis.yml]: You could say the same about coveralls, no?

Comment (josegonzalez) [inline .travis.yml]: At the end of the day, the php version for coding standards shouldn't matter imo.

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+src_dir: .
```
Comment (ADmad) [inline .coveralls.yml]: Coverage reporting on coveralls.io seems to works just fine with this. I don't really mind adding this file but would prefer if we don't add extra meta data files if not really needed.

File: .travis.yml (hunk 1 above)
Comment (ADmad) [inline .travis.yml]: Using php 7 helps see build results faster :)  That's enough incentive for me to use php 7.

Comment (lorenzo) [inline .travis.yml]: Exactly the version does not matter there, it is just a lot faster. Unfortunately the php version usually does matter for the coverage.

File: .coveralls.yml (hunk 2 above)
Comment (josegonzalez) [inline .coveralls.yml]: If that's the case, I'd also like to get rid of the `.semver` file we seem to add everywhere.

Comment (lorenzo) [inline .coveralls.yml]: :+1:

Comment (dereuromark) [inline .coveralls.yml]: :+1:

File: .travis.yml (hunk 1 above)
Comment (josegonzalez) [inline .travis.yml]: So coveralls breaks under php 7 but phpcs is like "yeah sure why not".

Computers are the fucking worst.

**Comment by dereuromark** [comment]
:+1:

File: composer.json [hunk 3]
```diff
@@ -14,20 +10,25 @@
...
             "homepage": "https://github.com/friendsofcake/bootstrap-ui/graphs/contributors"
         }
     ],
-    "support": {
-        "issues": "http://github.com/friendsofcake/bootstrap-ui/issues",
-        "source": "http://github.com/friendsofcake/bootstrap-ui"
+    "require": {
+        "cakephp/cakephp": "~3.0",
+        "php": ">=5.4.0",
+        "cakephp/plugin-installer": "*"
```
Comment (ADmad) [inline composer.json]: This is not needed. Plugin installer is required by app.

**Pull request merged**